docs/Model_Assumptions.rst

```diff
@@ -176,12 +176,117 @@ The confersion factors are derived from their `Gasoline Gallon Equivalents.<http
+#################################
+:Limitation: There is random excess distribution between the feed-in sink and the excess sink when no feed-in-tariff is assumed in the system.
+:Reason: Since there is no feed-in-tariff to benefit from, the MVS randomly distributes the excess energy between the feed-in and excess sinks.
+:Implications: This distribution of excess energy changes when running several simulations for the same input files.
+
+Renewable energy share defintion
+################################
+:Limitation: The current renewable energy share depends on the share of renewable energy production assets directly feeding the load.
+:Reason: The MVS tool is a work in progress and this can still be addressed in the future.
+:Implications: This might result in different values when comparing them to other models.
+
+Energy carrier rating
+#####################
+:Limitation: The MVS assumes a usable potential/energy content rating for every energy carrier.
+:Reason: This is an approach that the MVS currently uses.
+:Implications: By weighing the energy carriers according to their energy content (Gasoline Gallon Equivalent (GGE)), the MVS might result in values that can't be directly assessed. Those ratings affect the calaculation of the levelized cost of the energy carriers, but also the minimum renewable energy share constraint.
+
+Energy shortage or grid interruption
+####################################
+:Limitation: The MVS assumes no shortage or grid interruption in the system.
+:Reason: 
+:Implications: Electricity shortages due to power cuts might happen in real life. The MVS currently omits this scenario which might implicate unreasonable results. If a system is self-sufficient but relies on grid-connected PV systems, the latter stop feeding the load if any power cuts occur and the battery storage systems might not be enough to serve the load (energy shortage).
+
+Generic cost definitions
+########################
+:Limitation: The MVS includes development costs, specific costs, O&M costs and disptach costs.
+:Reason: The MVS assumes those costs only for simplification.
+:Implications: This could lead to a rough estimation of the entire project costs.
+
+Unlinked one technical unit transformers
+########################################
+:Limitation: Some components should be modelled as separate transformers in the MVS while in real life they are one technical unit.
+:Reason: The MVS uses oemof-solph's generic components listed here below:
+
+    - Sink (one input, no output)
+    - Source (one output, no input)
+    - Linear Transformer (one input, n outputs)
+    - Storage (one input, one output)
+:Implications: Since one input is only allowed, such technical units are modelled as two separate transformers that are currently unlinked in the MVS (e.g., hybrid inverter, heat pump, distribution transformer, etc.). This means that the costs (and sometimes capacities) of those transformers are either replicated or divided between them.
```

Comment:
```suggestion
:Implications: 

Since one input is only allowed, such technical units are modelled as two separate transformers that are currently unlinked in the MVS (e.g., hybrid inverter, heat pump, distribution transformer, etc.). This raises a difficulty to define costs in the input data. It also results in two optimized capacities for one logical unit.

This limitation is to be addressed with a constraint which links both capacities of one logical unit, and therefore solves both the problem to attribute costs and the previously differing capacities.
```